app/serializers/aichi11_target_dashboard_serializer.rb

```diff
@@ -89,7 +89,10 @@ def sorted
     _data = ActiveRecord::Base.connection.execute(query)
 
     if @params[:sort_by].present?
-      _data = _data.sort_by { |d| d[sort_by] }
+      sort_terms = sort_by.split('+')
+      _data = _data.sort_by do |d|
+        sort_terms.inject(0) { |sum, x| sum + (d[x] || 0) }
+      end
```

Comment:
I'm getting this error now while trying to access the dashboard, I think because of the `params[:sort_by]` being present by default with value `name`
![Screenshot from 2019-10-02 14-32-24](https://user-images.githubusercontent.com/40596660/66048648-d2831c00-e521-11e9-926d-b3682526fa09.png)

Reply:
That should be fixed now, thanks @lucacug !